libs/langchain/langchain/__init__.py

```diff
@@ -1,8 +1,12 @@
 # ruff: noqa: E402
 """Main entrypoint into package."""
+from pkgutil import extend_path
+__path__ = extend_path(__path__, __name__)
```

Comment:
Why do we need this? Feels like a hack that importing our package modifies the global python path for the entire process